README.md

```diff
@@ -1,3 +1,13 @@
 # sdk-link
 
-WIP
+## <a name='create-company'>Create a new company</a>
```

Comment:
I assume this is so you can link to it? I would normally expect that to be achieved like this:

```suggestion
<h2 id='create-company'>Create a new company</h2>
```

Reply:
Great, thanks for showing alternative ways of doing it. Used to do it with a tags in personal projects during GA, but will follow suggested pattern moving forward.